ccxcon/urls.py

```diff
@@ -18,4 +18,5 @@
     url(r'^admin/', include(admin.site.urls)),
     url(r'^api/v1/', include(router.urls)),
     url(r'^api/v1/', include(modules_router.urls)),
+    url(r'^api/v1/user_exists', 'courses.views.user_existence'),
```

Comment:
Can you add a trailing slash for consistency? I don't have a strong opinion either way but it would match the default behavior of Django REST framework. Also please add a trailing '$' to prevent anything else from matching